api/controllers/apps.go

```diff
@@ -107,27 +110,28 @@ func AppDelete(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 
 func AppLogs(ws *websocket.Conn) *httperr.Error {
 	app := mux.Vars(ws.Request())["app"]
+	header := ws.Request().Header
 
-	a, err := models.GetApp(app)
-
-	if awsError(err) == "ValidationError" {
-		return httperr.Errorf(404, "no such app: %s", app)
+	follow := true
```

Comment:
Yes for reporting. Particularly now that you can specify a time range. 

--since 24h --follow=false | grep 500 | wc